doc/tutorials/cpu_sharing.rst

```diff
@@ -99,6 +100,21 @@ and BVT (Borrowed Virtual Time) scheduler.
   tries to keep resources busy, and will run once it is ready. The idle thread
   can run when the vCPU thread is blocked.
 
+- **IO sensitive Round Robin scheduler**:
+
+  The IORR (IO sensitive Round Robin) scheduler supports multiple vCPUs running
+  on one pCPU, scheduled by a IO sensitive round robin policy.
+
+- **Priority based scheduler**:
+
+  The priority based scheduler can support vCPU scheduling based on their
+  pre-configured priorities. A vCPU can be running only if there is no
+  higher priority vCPU running on the same pCPU. For example, in some cases,
+  we have two VM, one VM can be configured to use **PRIO_LOW** and the
+  other one to use **PRIO_HIGH**, then the vCPU of the **PRIO_LOW** VM can
+  only be running when the vCPU of the "PRIO_HIGH" VM voluntarily relinquish
+  the usage of the pCPU.
+
```

Comment:
@ssqre Because of your comment in #6591, we should delete the TODO content on lines 124-127 below, right?

Reply:
I do agree to remove the TODO content in BVT doc. We don't have any plan to support any priority mechanism for BVT at this moment. It's better to update the BVT doc directly once we update any extension for BVT.